receiver/elasticapmreceiver/receiver.go

```diff
@@ -248,7 +260,8 @@ func (r *elasticAPMReceiver) processBatch(ctx context.Context, batch *modelpb.Ba
 				}
 			}
 		case modelpb.ErrorEventType:
-			// TODO
+			rl := ld.ResourceLogs().AppendEmpty()
```

Comment:
@gregkalapos quick question related to:
```go
	rl := ld.ResourceLogs().AppendEmpty()
        // [...]
	sl := rl.ScopeLogs().AppendEmpty()
	l := sl.LogRecords().AppendEmpty()
```

Given `processBatch` takes in `modelpb.Batch` which may contain many events of type ErrorEventType about the same Resource and even same Scope, did you consider building a plog.Logs that groups LogRecords under the same Scope and Resource?
I don;t think it matters much for the final destination, but could help processors that work on batches and iterate over the two sub "collections".